pkg/scheduler/framework/plugins/nodename/node_name.go

```diff
@@ -37,6 +39,14 @@ const (
 	ErrReason = "node(s) didn't match the requested node name"
 )
 
+// EventsToRegister returns the possible events that may make a Pod
+// failed by this plugin schedulable.
+func (pl *NodeName) EventsToRegister() []framework.ClusterEvent {
+	return []framework.ClusterEvent{
+		{Resource: framework.Node, ActionType: framework.Add | framework.Delete | framework.UpdateNodeAllocatable | framework.UpdateNodeLabel},
```

Comment:
- framework.Delete Does a deletion event help to make a unscheduleable Pod scheduable?
- framework.UpdateNodeAllocatable Does resource allocatable event help to make a unscheduleable Pod scheduable?

I doubt both of them, but let's hold and see what is @Huang-Wei thinking about this.